src/pallets/static/style.css

```diff
@@ -1,5 +1,10 @@
 header {
   border-bottom: 1px solid var(--pico-muted-border-color);
+  margin-bottom: 2.25rem;
+}
+
+header nav a {
+  text-decoration: none;
```

Comment:
I'd like to keep this, I think it's good to have some feedback when hovering links.

Reply:
Navigation menus are well understood these days, the contrast hover feedback I think is enough, along with the positioning to tell the user the items are clickable. Although, I do agree that the underline is a louder indication of being clickable.